pkgs/by-name/sy/syncyomi/package.nix

```
@@ -0,0 +1,106 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, buildGoModule
+, jq
+, moreutils
+, nodePackages
+, cacert
+, esbuild
+}:
+
+buildGoModule rec {
+  pname = "syncyomi";
+  version = "0.3.8";
+
+  src = fetchFromGitHub {
+    owner = "SyncYomi";
+    repo = "SyncYomi";
+    rev = "v${version}";
+    hash = "sha256-wufdJkQCJD/D7kd1TSkEKRz+1Ov5lAQUsvwrCluoHIw=";
+  };
+
+  vendorHash = "sha256-/rpT6SatIZ+GVzmVg6b8Zy32pGybprObotyvEgvdL2w=";
+
+  pnpmDeps = stdenvNoCC.mkDerivation {
+    pname = "${pname}-pnpm-deps";
+    inherit src version;
+    sourceRoot = "${src.name}/web";
+
+    nativeBuildInputs = [
+      jq
+      moreutils
+      nodePackages.pnpm
+      cacert
+    ];
+
+    installPhase = ''
+      runHook preInstall
+
+      export HOME=$(mktemp -d)
+      pnpm config set store-dir $out
+      pnpm install --frozen-lockfile --no-optional --ignore-script
+
+      rm -rf $out/v3/tmp
+      for f in $(find $out -name "*.json"); do
+        sed -i -E -e 's/"checkedAt":[0-9]+,//g' $f
+        jq --sort-keys . $f | sponge $f
+      done
+
+      runHook postInstall
+    '';
+
+    dontBuild = true;
+    dontFixup = true;
+    outputHashMode = "recursive";
+    outputHash = "sha256-YOmAgzs3tDvZLRgFLkj6CY88Fm5fxbpqCUhILc8n+6c=";
+  };
+
+  nativeBuildInputs = [
+    nodePackages.pnpm
+  ];
+
+  ESBUILD_BINARY_PATH = lib.getExe (esbuild.override {
```

```suggestion
  env.ESBUILD_BINARY_PATH = lib.getExe (esbuild.override {
```

being explicit about which attrs are used just as envvars (rather than actual attrs) is helpful for anyone reading the code